### PR TITLE
Add hypothetical meat consumption

### PR DESCRIPTION
**Pull request by pabloarosado**
Add a table to the dataset of additional FAOSTAT variables for the hypothetical meat production that would be required if the world ate like the average citizen of a given country.

I have also added a similar variable for the number of slaughtered animals.

### CODE REVIEW

File: etl/steps/data/garden/faostat/2023-02-22/additional_variables.meta.yml [hunk 1]
```diff
@@ -626,3 +626,45 @@ tables:
...
+        unit: "animals"
+        short_unit: ""
+      animals_global_hypothetical:
+        title: Hypothetical number of slaughtered animals if everyone ate like the average citizen of a given country
+        unit: "animals"
+        short_unit: ""
+        description: |
+          Hypothetical number of slaughtered animals worldwide if everyone in the world ate the same quantity as the average citizen of a given country.
+
+          This is a hypothetical variable derived by Our World in Data which answers the question: "how many animals would need to be slaughtered if everyone in the world was to consume the average per capita amount of a given country?". For example "how many animals would need to be slaughtered if everyone in the world consumed the same amount of meat as the average UK citizen?".
```
Comment (lucasrodes) [inline etl/steps/data/garden/faostat/2023-02-22/additional_variables.meta.yml]: Unsure about it, but should the question start with caps? 

> "How many...

**Review by lucasrodes — APPROVED** [review]
LGTM! Thanks for doing this, Pablo. Minor comments.

File: etl/steps/data/garden/faostat/2023-02-22/additional_variables.meta.yml (hunk 1 above)
Comment (pabloarosado) [reply etl/steps/data/garden/faostat/2023-02-22/additional_variables.meta.yml]: I agree, done.

File: etl/steps/data/garden/faostat/2023-02-22/additional_variables.py [hunk 2]
```diff
@@ -927,6 +927,98 @@ def generate_agriculture_land_evolution(df_rl: pd.DataFrame) -> Table:
...
+    ITEM_CODE_FOR_MEAT_TOTAL = "00001765"
+
+    # Select the required items/elements/units to get national data on per-capita production and slaughtered animals.
+    meat = df_qcl[
+        (df_qcl["item_code"] == ITEM_CODE_FOR_MEAT_TOTAL)
+        & (df_qcl["element_code"].isin([ELEMENT_CODE_FOR_PRODUCTION_PER_CAPITA, ELEMENT_CODE_FOR_ANIMALS_PER_CAPITA]))
+        & (df_qcl["unit"].isin([UNIT_FOR_PRODUCTION_PER_CAPITA, UNIT_FOR_ANIMALS_PER_CAPITA]))
+    ].reset_index(drop=True)
+    meat = meat.pivot(index=["country", "year"], columns=["element_code"], values=["value"])
+    meat.columns = [column[1] for column in meat.columns]
```
[the remark this answers is not in the document]
Comment (pabloarosado) [reply etl/steps/data/garden/faostat/2023-02-22/additional_variables.py]: Nice one! Every time I use pivot I encounter these kinds of issues. What you suggested solves the problem in this case, although the index is still renamed to "element_code", which is quite a spurious behaviour, in my opinion. At some point maybe we could host a pivot function in `datautils` that deals with these minor inconveniences.

**Pull request merged**